backend/src/routes/auth.ts

```diff
@@ -0,0 +1,19 @@
+import { NextFunction, Request, Response, Router } from 'express';
+import passport from 'passport';
+import User from 'models/user';
+
+const router = Router();
+
+const handleLogin = (req: Request, res: Response, next: NextFunction) => {
+  const user = req.user as User;
+  req.login(user, { session: false }, async error => {
```

Comment:
I do not believe this part is necessary since the user is already authenticated through the `passport.authentication` middleware. What is the reasoning behind this?